src/call_wrapper.ts

```diff
@@ -44,25 +44,27 @@ class CallWrapper {
         }
     }
 
-    private static calculateInitialMemorySize(parameters: any[]): number {
+    private static calculateInitialMemorySize(parameters: any[], initialMemorySize: number): number {
```

Comment:
Make method non-static and use the `initialSize` field of the `CallWrapper` instead of passing it as a parameter.